interface/super/edit_layout.php

```diff
@@ -565,7 +565,7 @@ class='optin noselect' style='width:100%' />";
 
         echo "<input type='text' name='fld[$fld_line_no][list_id]'  id='fld[$fld_line_no][list_id]' value='" .
         htmlspecialchars($linedata['list_id'], ENT_QUOTES) . "'".$type.
-        " size='6' maxlength='30' class='optin listid' style='width:100%;cursor:pointer'".
+        " size='6' maxlength='100' class='optin listid' style='width:100%;cursor:pointer'".
```

Comment:
I believe this attribut determines the maximum number of input characters the field can accept,  of the actual max integer. So the number `100` only needs a `maxlength` of `3`